src/components/pets/PetType.js

```diff
@@ -212,6 +220,12 @@ export default function PetType({ token }) {
                       src={img}
                       onMouseEnter={onHoverPhoto}
                       onMouseLeave={onBlurPhoto}
+                      style={{
```

Comment:
can you put this part in css in `pets.css`?
you could put it like
 ```
 .petList__container .card-img-top {
  //css here
}
```

Reply:
Should be changed in the next commit